modules/apps/layout/layout-reports-web/jest-setup.config.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+window.Liferay.Util.sub = function (string = '', data) {
+	const REGEX_SUB = /(?<=-|^)x(?=-|\s)/g;
+
+	if (
+		arguments.length > 2 ||
+		(typeof data !== 'object' && typeof data !== 'function')
+	) {
+		data = Array.prototype.slice.call(arguments, 1);
+	}
+
+	const dataCopy = [...data];
+	const max = REGEX_SUB.exec(string).length;
+	let replacedValues = 0;
+
+	const replacestring = string.replace
```

Comment:
Another non-blocking mini-nit: `replacestring` → `replaceString`

Reply:
Also here...Hopefully, this file will be removed soon and provide it by default in our tests 🍀